src/kili/mutations/label/__init__.py

```diff
@@ -158,12 +164,14 @@ def append_to_labels(
     @typechecked
     def append_labels(
         self,
-        asset_id_array: List[str],
         json_response_array: List[Dict],
         author_id_array: Optional[List[str]] = None,
         seconds_to_label_array: Optional[List[int]] = None,
         model_name: Optional[str] = None,
         label_type: LabelType = "DEFAULT",
+        project_id: Optional[str] = None,
+        asset_id_array: Optional[List[str]] = None,
```

Comment:
The thing is that we cannot put argument with default value in front of arguments without default values...

Reply:
oh yes I missed the `json_response_array` sorry. This is a breaking change then. A way to circumvent that is to make `json_response_array` a keyword argument, this would allow us to leave asset_id_array at the same place. Wdyt ?

Reply:
or :
```
        asset_id_array: Optional[List[str]],
        json_response_array: List[Dict],
        author_id_array: Optional[List[str]] = None,
        seconds_to_label_array: Optional[List[int]] = None,
        model_name: Optional[str] = None,
        label_type: LabelType = "DEFAULT",
        project_id: Optional[str] = None,
```

Reply:
I went for the json_response_array to have `[]` as default value. It makes things not breaking and does not change the UX for the client. We can change that when we will do the general ticket for asset_id/external_id

Reply:
This should be:
```
  asset_id_array: Optional[List[str]] = None,
  json_response_array: List[Dict] = [],
  author_id_array: Optional[List[str]] = None,
  seconds_to_label_array: Optional[List[int]] = None,
  model_name: Optional[str] = None,
  label_type: LabelType = "DEFAULT",
  project_id: Optional[str] = None,
  asset_external_id_array: Optional[List[str]] = None,
 ```
  right?

Reply:
otherwise this is breaking

Reply:
Oh yeah, mistake!